Resources/Prototypes/Entities/Markers/Spawners/jobs.yml

```diff
@@ -36,15 +36,18 @@
 
 # Cargo
 
-#- type: entity
-#  id: SpawnPointQuartermaster
-#  parent: SpawnPointJobBase
-#  name: quartermaster
-#  components:
-#    - type: SpawnPoint
-#      job_id: Quartermaster
-#    - type: Sprite
-#      state: Quartermaster
+- type: entity
+  id: SpawnPointQuartermaster
+  parent: SpawnPointJobBase
+  name: quartermaster
+  components:
+  - type: SpawnPoint
+    job_id: Quartermaster
+  - type: Sprite
+    layers:
+      - state: green
+      - state: qm
```

Comment:
i feel like i remember trying to readd this on my fork and it errored because this state doesnt exist, does it exist now

Reply:
Yep, someone must of added a bunch of job sprites. Even the psychologist is in there now.